sse2neon.h

```diff
@@ -3643,6 +3643,23 @@ FORCE_INLINE int64_t _mm_popcnt_u64(uint64_t a)
 #endif
 }
 
+// Macro: Transpose the 4x4 matrix formed by the 4 rows of single-precision
+// (32-bit) floating-point elements in row0, row1, row2, and row3, and store the
+// transposed matrix in these vectors (row0 now contains column 0, etc.).
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=MM_TRANSPOSE4_PS&expand=5949
+#define _MM_TRANSPOSE4_PS(row0, row1, row2, row3) \
+    do {                                          \
```

Comment:
If the following function produce the exact semantics of SSE intrinsics, we don't have to validate macro `_MM_TRANSPOSE4_PS`.

Reply:
I think I will remove the part of the validation.

Reply:
Done.